### ISSUE TITLE
"Expected more than 1 value per channel when training".

### ISSUE THREAD

**Issue by Bonsen** [fastai/fastai#1352]
Sorry, I can't find where to create a new topic on forum.... So I ask here..

I use the senet154 from "fastai/old/fastai/models/senet.py" on fastai '1.0.22'.

data = ImageDataBunch.create(train_ds, val_ds, test_ds=test_ds, path=path, bs=bs, tfms=(tfms, []),  num_workers=8, size=512).normalize(kk)
learn = create_cnn(
    data,
    senet154,
    ps=0.5,
    cut=-2,
    path=path,    
    metrics=[acc]
)
learn.model = nn.DataParallel(learn.model)
learn.callback_fns.append(partial(SaveModel, every='improvement', monitor='val_loss'))
learn.fit_one_cycle(5, lrs) 
After 1 epoch end, it gives:
"Expected more than 1 value per channel when training".

And if I use size=128 or 256 of ImageDataBunch.create for res50, everything is fine.
If the small batchsize and the number of training set cause this? Is there anything like "droplast" of pytorch in fastai?

**Comment by sgugger** [comment]
That's because you didn't use `drop_last=True` for your training dataloader (now default in fastai) and ran in a batch of size 1, which causes error with BatchNorm during training (that's not us, it's on pytorch). You should use that option (or update your library).
